.gitignore

```diff
@@ -5,6 +5,8 @@
 # macOS
 .DS_Store
 
+**/*.orig
```

Comment:
`*.orig` will capture files at the base of the project directory, so adding `**/` in front of that will ensure that any files no matter their parent directory should be ignored (e.g. com/some/package/SomeClass.scala.orig)